addons/payment_mollie_official/models/payment_acquirer.py

```diff
@@ -0,0 +1,172 @@
+        return self.update_payment_icon_ids()
+
+    def _get_mollie_urls(self, state):
+        """ Mollie URLS """
+        url = {
+            'prod': 'https://api.mollie.com/v2',
+            'test': 'https://api.mollie.com/v2', }
+
+        return {'mollie_form_url': url.get(state, url['test']), }
+
+    def mollie_form_generate_values(self, values):
+        self.ensure_one()
+        base_url = self.env['ir.config_parameter'].sudo(
+        ).get_param('web.base.url')
+        mollie_api_key = self._get_mollie_api_keys(
+            self.state)['mollie_api_key']
+        mollie_tx_values = dict(values)
+        order_id = request.session.get('sale_last_order_id')
+        mollie_tx_values.update({
+            'OrderName': values.get('reference'),
+            'Description': values.get('reference'),
+            'Amount': '%.2f' % float(values.get('amount')),
+            'Currency': values['currency'] and values['currency'].name or '',
+            'Key': mollie_api_key,
+            'URL': self._get_mollie_urls(self.state)['mollie_form_url'],
+            'BaseUrl': base_url,
+            'Language': values.get('partner_lang'),
+            'Name': values.get('partner_name'),
+            'Email': values.get('partner_email'),
+            'Zip': values.get('partner_zip'),
+            'Address': values.get('partner_address'),
+            'Town': values.get('partner_city'),
+            'Country': values.get('partner_country') and
+            values.get('partner_country').code or '',
+            'Phone': values.get('partner_phone'),
+            'webhookUrl': base_url,
+            'OrderId': order_id,
+        })
+        return mollie_tx_values
+
+    def mollie_get_form_action_url(self):
+        self.ensure_one()
+        return "/payment/mollie/intermediate"
+
+    def update_payment_icon_ids(self):
+        self.ensure_one()
+        if self.provider != 'mollie':
+            return
+        icon_model = self.env['payment.icon']
+        icon_ids = []
+        for method in self.method_ids:
+            icon = icon_model.search([
+                ('acquirer_reference', 'ilike', method.acquirer_reference)],
+                limit=1)
+            if not icon:
+                icon = icon_model.create({
+                    'name': method.name,
+                    'acquirer_reference': method.acquirer_reference,
+                    'image': method.image_small,
+                    'sequence': method.sequence,
+                    'provider': self.provider,
+                    'currency_ids': method.currency_ids,
+                    'country_ids': method.country_ids
+                })
+                icon.onchange_provider_ref()
+            icon_ids.append(icon.id)
+            icon.write({'sequence': method.sequence,
+                        'provider': self.provider})
+
+        return self.update({'payment_icon_ids': [(6, 0, icon_ids)]})
+
+    def update_available_mollie_methods(self):
+        for acquirer in self:
```

Comment:
I don't want any of this. Users can set their payment icons manually, based on what they configured in the mollie backend. Synchronization just for this is overkill imho.

Reply:
this will also remove the cron, the ´write´ override and a lot of this code